_posts/2025-02-25-the-case-for-fedora-flatpaks.md

```diff
@@ -0,0 +1,92 @@
+* All applications that we ship (regardless of packaging method) are FOSS and legally unencumbered.
+* We actively work to move all applications to the latest runtime version as quickly as possible, and work with any individual package maintainers to address any issues that prevent doing so.
+* All packages use the same version of non-runtime dependencies.  All dependents benefit equally from any given fix or update to a dependency.  Specifically in the context of Fedora Flatpaks, not only does this simplify and speed up builds, but these will actually deduplicate on-disk because they are the very same builds packed with each dependent application.
+* All Fedora packaging is part of a single community.  Even though SIGs or individual maintainers may focus on certain parts of the distribution, there is a single means for submitting contributions (Pull Requests on src.fedoraproject.org).
+
+Fedora Flatpaks simply serve as an additional packaging format for our builds of desktop applications.  Because Fedora features several editions and spins which are ostree-based ("Atomic" or "immutable" systems), Fedora Flatpak content is particularly needed to allow those users to install Fedora content in a way that is most compatible with that format (where adding packages to the host requires various additional steps.)  Even users of traditional systems (such as myself) can still benefit from Flatpaks over RPMs, and if this is indeed the future of desktop applications, then all users are better served by a robust selection of software in this format.
+
+Furthermore, only by packaging a broad selection of applications as Fedora Flatpaks have issues and limitations of our packaging, runtimes, and tooling been discovered, which then makes it possible to fix and improve them, benefiting the entire ecosystem.  Not only will users of Fedora Flatpaks benefit from these changes, but sometimes Fedora RPM users as well.  These improvements should then eventually filter down to the RHEL Flatpak runtime and SDK.  (Some changes are already on track for RHEL 10.0, with more hopefully to follow.)
+
+Even Flathub itself, and the upstreams who ship their own software thereon, could end up benefiting from robust alternatives.  There are plenty of individual or systemic issues which I have discovered in Flathub Flatpaks only by virtue of working on Fedora Flatpaks, and they too could benefit if there were reliable means to communicate them.
+
+In summary, users still want distributions of software, choose a distribution by varying factors, and expect their distribution to meet its stated goals and standards.  Fedora Flatpaks simply serve the next step in the Fedora distribution with respect to desktop applications in this era of new desktop and system technologies.
+
+### How To Participate
+
+If you are genuinely interested in the future of Fedora Flatpaks, then there are many ways in which you can become involved:
+* Automation
```

Comment:
Hello!

In [Packit](https://packit.dev/), we are trying to help automate various things across Fedora so maybe we can discuss if we can't help here somehow as well. (TBH, I don't want to give any false promises since this is quite a small use case compared to our other efforts and we have limited team capacity, but I hope we can figure something out.)

We can set up a quick call to discuss this if you want...